src/client/public/static/excel/instructions.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Reactive Trader Excel: Installation instructions</title>
+    <style>
+      body {
+        font-family: 'Lato', 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+          'Helvetica Neue', sans-serif;
+        background-color: #2e3543;
+        color: white;
+      }
+      article {
+        padding: 0 10px;
+      }
+      a,
+      a:active,
+      a:visited,
+      a:link,
+      a:hover {
+        color: #a8c9fe !important;
+      }
+    </style>
+  </head>
+
+  <body>
+    <article>
+      <h1>
+        Getting Started with Reactive Trader Excel
+      </h1>
+      <p>
+        To use
+        <a
+          href="https://weareadaptive.com/2016/10/05/rich-desktop-experience-openfin/"
+          target="_blank"
+          >ReactiveTraderExcel</a
+        >
+        you need to follow these steps.
+      </p>
+      <p>
+        Download the the zip file
+        <a href="../../plugin/ReactiveTraderExcel.zip">from here</a>.
+      </p>
+      <p>Don't extract the zip file yet.</p>
+      <p>Right click, select properties, check "Unblock" and press "ok".</p>
```

Comment:
Could you please specify which property of the file we have to check "Unblock"?

Reply:
To be honest this html file is an exact replica of [`docs/setup/reactive-trader-excel.md`](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/develop/docs/setup/reactive-trader-excel.md) so haven't paid much attention to the actual contents! I managed to follow it to make it work though.

The 'unblock' it refers is this one on the properties dialog:

![image](https://user-images.githubusercontent.com/4398367/55727795-aa9b7b80-5a0a-11e9-88a2-1d338c25d373.png)

I think it may not be always visible, maybe only when downloading from the internet. Ralph couldn't see the checkbox after downloading the .zip file when running on localhost (as the .zip is downloaded from localhost as well)